shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -89,6 +96,14 @@ private void init() {
   @TargetApi(19)
   @NonNull
   private static ImageReader createImageReader(int width, int height) {
+    if (width <= 0) {
+      Log.w(TAG, "ImageReader dimensions must > 0, but given width=" + width + ", set width=1");
+      width = 1;
+    }
+    if (height <= 0) {
+      Log.w(TAG, "ImageReader dimensions must > 0, but given height=" + height + ", set height=1");
```

Comment:
```suggestion
      Log.w(TAG, "ImageReader height must be must greater than 0, but given height=" + height + ", set height=1");
```

Reply:
I want to put it to one line, otherwise it will looks like:
```java
    if (width <= 0) {
      Log.d(
          TAG,
          "ImageReader width must be greater than 0, but given width=" + width + ", set width=1",
          width);
      width = 1;
    }
```
So I extract a log method:
```java
    if (width <= 0) {
      logW("ImageReader width must be greater than 0, but given width=%d, set width=1", width);
      width = 1;
    }
```
It looks good now!  @blasten Please review again~